cmd/aries-mobile-sdk/main.go

```diff
@@ -0,0 +1,11 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	//"github.com/hyperledger/aries-framework-go/cmd/aries-mobile-sdk/pkg/framework"
```

Comment:
You are adding go.sum and go.mod  files but the main file is empty. Please add at least one import. e.g
`_ "github.com/hyperledger/aries-framework-go/cmd/aries-mobile-sdk/pkg/framework"`

Reply:
i'll use this `_ "github.com/hyperledger/aries-framework-go"` thanks